data/extra.csv

```diff
@@ -20,9 +20,9 @@ Hope's Scarf (XIII),Accessory,4,,15,,,,,,,15,,,,,
 Inferno Fang (VIII),Dagger,5,64,,95,,,,,113,,95,,,,
 Cat's Claws (IX),Fist,5,77,,95,,,,,126,,95,,,,
 Heavy Coat (XII),Light Armor,3,,,,20,21,99,,,,,48,48,99,
-Witch's Hat (XIII),Hat,3,,18,,36,70,,15,,24,,51,100,,21
+Witch's Hat (XIII),Hat,5,,18,,36,70,,15,,24,,51,100,,21
```

Comment:
Ah, I see. It's because these are special items, so even though they're 5* items, the max stats are for level 20, not level 30.

So what to do I do now? I'm guessing I should change these back to 3* in my own branch, and then send you a new pull request?